### DISCUSSION TITLE
How can I try kepler on a VM?

### DISCUSSION THREAD

**Discussion by mincheolsu** [sustainable-computing-io/kepler#922] [Q&A]
Hi,

I have a fedora VM and installed Kepler through the RPM package.
Then, how can I install Grafana or Prometheus to see the power metric?
Can I follow https://sustainable-computing.io/installation/kepler/#deploy-the-prometheus-operator?

**Comment by husky-parul** [comment]
Hi @mincheolsu 
The above document is for installing Prometheus within a cluster. Since you've used the Kepler rpm I would assume you are running this on a VM based linux server. We have not yet documented how to deploy Prom on a linux server but this doc explains the steps
https://devopscube.com/install-configure-prometheus-linux/

**Comment by rizwanrasheed22** [comment]
Is Kepler package available for Debian/Ubuntu ?
